#!/usr/bin/env python
PACKAGE = "ppdr_filter_tools"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("forget_factor", double_t, 0, "Forget factor", 0.5, 0, 1)
gen.add("lpf_cutoff_freq", double_t, 0, "LPF cutoff frequency", 10, 0, 100)
gen.add("lpf_sampling_freq", double_t, 0, "LPF sampling frequency", 50, 0, 1000)

exit(gen.generate(PACKAGE, "ppdr_filter_tools", "filter_param"))